app/src/main/java/com/example/myhouse_aos/presentation/content/adapter/ItemContentDetailAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.myhouse_aos.presentation.content.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.myhouse_aos.R
+import com.example.myhouse_aos.databinding.ItemContentDetailBinding
+import com.example.myhouse_aos.domain.model.ContentDetailData
+import com.example.myhouse_aos.util.extension.ItemDiffCallback
+import com.example.myhouse_aos.util.snackbar.ScrapSnackBar
+
+class ItemContentDetailAdapter : ListAdapter<ContentDetailData.Images, ItemContentDetailAdapter.ItemContentDetailViewHolder>(
+    ItemDiffCallback<ContentDetailData.Images>(
+        onItemsTheSame = {old, new -> old.image_id == new.image_id},
+        onContentsTheSame = {old, new -> old == new}
+    )
+)
+{
+    class ItemContentDetailViewHolder(private val binding: ItemContentDetailBinding) :
+            RecyclerView.ViewHolder(binding.root){
+                fun onBind(data: ContentDetailData.Images){
+                    binding.data = data
+                    if(data.content == null)
+                    {
+                        binding.tvItemContentdetail.visibility = View.GONE
+                    }
+
+                    var isButtonPressed = false
+
+                    binding.btnItemContentdetailScrap.setOnClickListener {
+                        if(isButtonPressed){
+                            isButtonPressed = false
```

Comment:
```suggestion
                if (!bookmark.isSelected) {
                    bookmark.isSelected = true
                    // TODO 스낵바 띄우기
                }
                else {
                    bookmark.isSelected = false
                    // TODO 스크랩 삭제 api 호출
                }
            }
```
isSelcted라는 속성이 있더라구용 이거 활용해 봐도 좋을거 같습니다!

Reply:
isSelected라는 속성이 있었군요 !

Reply:
저도 처음알았습니다! !!